Dockerfile

```diff
@@ -25,33 +26,22 @@ RUN if [[ "$(uname -m)" -eq "x86_64" ]] ; then \
          echo "CPU architecture not supported" && exit 1; \
      fi
 
-# Install oc binary
-ENV OC_BIN_TAR="openshift-client-linux.tar.gz"
-ENV OC_DL_URL="https://mirror.openshift.com/pub/openshift-v4/clients/ocp"/${OPENSHIFT_VERSION}/${OC_BIN_TAR}
-ENV OC_BIN="/usr/local/oc/bin"
-RUN wget --directory-prefix=${TEMP_DIR} ${OC_DL_URL} && \
-    mkdir -p ${OC_BIN} && \
-    tar -C ${OC_BIN} -xzf ${TEMP_DIR}/${OC_BIN_TAR} && \
-    chmod a+x ${OC_BIN}/oc
-
-# Add go and oc binary directory to $PATH
 ENV PATH=${PATH}:"/usr/local/go/bin":${GOPATH}/"bin"
 
-# Git identifier to checkout
-ARG TNF_VERSION
-ARG TNF_SRC_URL=$TNF_SRC_URL
-ARG GIT_CHECKOUT_TARGET=$TNF_VERSION
+# Download operator-sdk binary
+ENV OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.26.0 \
+    OSDK_BIN="/usr/local/osdk/bin"
+RUN mkdir -p ${OSDK_BIN} && \
+    curl --location --remote-name ${OPERATOR_SDK_DL_URL}/operator-sdk_linux_amd64 && \
+    mv operator-sdk_linux_amd64 ${OSDK_BIN}/operator-sdk && \
+    chmod +x ${OSDK_BIN}/operator-sdk
 
-# Clone the TNF source repository and checkout the target branch/tag/commit
-RUN git clone --no-single-branch --depth=1 ${TNF_SRC_URL} ${TNF_SRC_DIR}
-RUN git -C ${TNF_SRC_DIR} fetch origin ${GIT_CHECKOUT_TARGET}
-RUN git -C ${TNF_SRC_DIR} checkout ${GIT_CHECKOUT_TARGET}
-
-# Build TNF binary
+# Copy all of the files into the source directory and then switch contexts
+COPY . ${TNF_SRC_DIR}
 WORKDIR ${TNF_SRC_DIR}
 
 RUN make install-tools && \
-	make build-cnf-tests-debug
+	make build-cnf-tests
```

Comment:
Caution: this will build the release version of the tnf tool, which means sometimes we might not get too much information from panics, due to the compiler optimizations. I'm ok with it, but I believe we should also provide a a way for users to run the tnf in debug mode, just in case. Can be done in further PRs, if agreed.

Reply:
Agreed, we can add a debug image in a future PR.